community/syslinux/patches/0018-prevent-pow-optimization.patch

```diff
@@ -0,0 +1,36 @@
+From: Lukas Schwaighofer <lukas@schwaighofer.name>
```

Comment:
This looks like a much nicer patch: https://git.archlinux.org/svntogit/packages.git/tree/trunk/0018-prevent-pow-optimization.patch?h=packages/syslinux